packages/bpk-component-rating/src/BpkRating.scss

```diff
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins';
+
+.bpk-rating {
+  display: flex;
+
+  &--base-rating {
+    width: $bpk-spacing-sm * 4;
```

Comment:
As we're already adding new tokens, why don't we define these there too so that they all share one source of truth?

E.g. in `ratings.json`:

```
    "RATING_WIDTH": {
      "value": "{!SPACING_SM} * 4",
      "type": "color"
    },
```

Then in this file you can just do:

```
width: $bpk-rating-width;
```

We don't need to do these for all things, e.g. `text-align` but where there are numbers or colours involved it's generally a good thing to do.